CougSat1-ADCS/src/drivers/IHUCOM/IHUCOM.cpp

```diff
@@ -0,0 +1,31 @@
+#include "IHUCOM.h"
+
+
+IHUCOM::IHUCOM(): IHU(PIN_I2C_BUS_SDA, PIN_I2C_BUS_SCL)
+{
+    IHU.address(0x99);
```

Comment:
should the address be `0xAC` (or is it `0xAD`)?

Reply:
The address is our testing address not the actual one. I will change it to the real one.